### DISCUSSION TITLE
RabbitMQ Erlang Version Requirements - is it up-to-date?

### DISCUSSION THREAD

**Discussion by inikulshin** [rabbitmq/rabbitmq-server#7728] [Questions] [answered]
https://www.rabbitmq.com/which-erlang.html

Specifically, I'm interested in `3.10.8` is really limited to Erlang OTP `25.2`?

@ophirzk FYI

**Comment by lukebakken** [comment, marked as answer]
`3.10.8` is outdated, so if you'd like to use Erlang 25.3, please upgrade to the latest `3.10.x` version.

> Specifically, I'm interested in 3.10.8 is really limited to Erlang OTP 25.2?

Probably not strictly but my guess is that we never tested `3.10.8` with Erlang `25.3` so you'd be on your own.
